src/components/home/displayCharacters.js

```diff
@@ -0,0 +1,234 @@
+    const commentsBtn = document.createElement('button');
+    commentsBtn.innerText = 'Comments';
+    commentsBtn.classList.add('commentsBtn');
+    commentsBtn.classList.add('btn');
+    const reservationsBtn = document.createElement('button');
+    reservationsBtn.classList.add('btn');
+    reservationsBtn.innerText = 'Reservations';
+    reservationsBtn.classList.add('reservationsBtn');
+    characterDiv.appendChild(commentsBtn);
+    characterDiv.appendChild(reservationsBtn);
+
+    character.occupation.forEach((occupation) => {
+      const occupationList = document.createElement('li');
+      occupationList.classList.add('occupationList');
+      occupationList.innerText = `Occupation: ${occupation}`;
+      charactersOccupation.appendChild(occupationList);
+    });
+    const counterChar = document.getElementById('CharacterCounter');
+    characterCounter(counterChar, charactersInfo.length);
+  });
+
+  // Comments Section
+  const close = document.querySelector('.close');
+  const container = document.querySelector('.general__container');
+
+  const btnComments = document.querySelectorAll('.commentsBtn');
+  const nameSpace = document.querySelector('.name__space');
+  const photo = document.querySelector('.photo__breaking');
+  const contOne = document.querySelector('.one');
+  const contTwo = document.querySelector('.two');
+  const contThree = document.querySelector('.three');
+  const contFour = document.querySelector('.four');
+  const addComment = document.querySelector('.name__add');
+  const input1 = document.querySelector('.input__text');
+  const input2 = document.querySelector('.input__mail');
+  const insideBtn = document.createElement('button');
+  insideBtn.classList.add('inside__button');
+  const newComment = document.createElement('div');
+  newComment.classList.add('newComment');
+  const general = document.querySelector('.general__container');
+  const newParagraph = document.createElement('p');
+  newParagraph.classList.add('text__paragraph');
+  newComment.appendChild(newParagraph);
+  general.insertBefore(newComment, addComment);
+
+  btnComments.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      const inputMain = document.querySelector('.input__main');
+      const idDinamic = btn.parentElement.id; // id dinamico get
+      const idPath = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments/`; // id dinamico post
+      const pathGet = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments/?item_id=${idDinamic}`;
+
+      container.style.display = 'block';
+
+      fetch('https://www.breakingbadapi.com/api/characters?limit=12&offset=0') // GET BREAKING BAD DATA
+        .then((res) => res.json())
+        .then((data) => {
+          data.forEach((data) => {
+            if (JSON.stringify(data.char_id) === btn.parentElement.id) {
+              nameSpace.innerText = data.name;
+              console.log(nameSpace);
```

Comment:
- [ ] for javascript best practices try always to not commit `console.log`  to your repo. It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website. kindly remove all of the console logs in your pages like displayLikes.js, passLike.js, etc ... 🙏